users/webauthn.rst

```diff
@@ -10,25 +10,24 @@ and must be served at exactly the address ``https://localhost:8384``,
 unless configured otherwise as described in :ref:`webauthn-custom-gui-address`.
 
 WebAuthn authentication will be enabled if you have at least one `credential`, also called a `passkey`, registered.
-A credential is a public-private key pair stored either on an external security key,
-or a `platform credential` stored on your computer or phone.
+A credential is a public-private key pair that is stored on an `authenticator`,
+which could be an external security key, a smartphone, or built into your computer.
```

Comment:
```suggestion
which could be a smart device (such as a phone, tablet, or watch), a dedicated hardware security key (such as a YubiKey), or built right into your computer.
```
"security key" might be too vague to the uninitiated, and aren't they all external?